src/DataTable/TableRow.js

```diff
@@ -50,12 +52,15 @@ const pointerCSS = css`
 
 const TableRowStyle = styled.div`
   display: flex;
+  flex-wrap: wrap;
   align-items: stretch;
   align-content: stretch;
   width: 100%;
   box-sizing: border-box;
-  min-height: ${props => (props.dense ? props.theme.rows.denseHeight : props.theme.rows.height)};
-  ${props => (props.theme.rows.spacing === 'spaced' ? spacedRowsCSS : defaultRowsCSS)};
+  min-height: ${props =>
```

Comment:
We should preserve the original formatting to keep not only to keep the diff clean but to be consistent with other components

Reply:
Also, we don't want to change this since this is not relevant to the PR